### DISCUSSION TITLE
dota 2 won't launch

### DISCUSSION THREAD

**Discussion by soultaker6** [ValveSoftware/Dota2-Gameplay#5368] [Q&A]
Hello guys

I'm having some kind of weird trouble with Dota 2 on steam. I installed new Linux cinnamon mint 21 Vanessa and after I installed steam. It was everything in order so far. But when I reinstalled my games from my Steam library and when Dota 2 was installed it wouldn't launch, no matter what I tried to do from you tube videos, on how to troubleshoot dota2 to launch, nothing helped. Before installing Linux cinnamon 21 I used Linux cinnamon mint 18 and everything worked as is should. Also other games work just fine even on current version (cinnamon 21). I'm turning to you if anybody has any kind of idea what could be wrong or if someone had similar problem and how he fixed it I would love a reply. 
Thank you for help.

Best regards Z

**Comment by soultaker6** [comment]
Right, that was the thing I was careful of and I updated all the drivers from Nvidia but there was no change. The game wouldn't launch even after the updated drivers for graphic card. I would like to find what is the reason for this launching failure on my current software setup. I doubt it that it's the new Linux version.

**Comment by A-gent** [comment]
I know this is somewhat of a necro reply but, are you still having this problem? And if you are, tell me exactly what happens when you try to launch it. Is there an error message?

**Comment by soultaker6** [comment]
It's not necro at all, I'm still having this issue. When i try to launch Dota 2 vulcan shaders starts, it updates the game if it's necessary and then nothing happens. In steam console when l am looking at my library it shows if the game updates and it says launching and then it says running in green letters for a short amount of time (0,5s or so). But the game actually doesn't launch. After this happens it goes back like it wasn't even launched. so basicly when l press launch it looks all good and like it's about to launch, but it's not. I've tried to verify integrity of game files, try to setup launching options in steam console, uninstalled the game and installed it all over again it's always the same thing. it won't launch and that's. It looks like l've been playing it for too long and this is the universe reaching to me that it's time to stop or something... would be still glad if you have any kind of idea for me to try.

**Comment by sellerdash09** [comment]
Does anyone have the same issue? Just got a 12mb update now and I can't open dota anymore[.](https://sellerdash.pk/) It crashes after it shows the dota logo. Please help me solve it.

**Reply by trichome3** [reply]
Yes. I witnessed the same[.](https://sellerdash.pk/) The latest update made it so it gets to the loading screen then instantly breaks. ubuntu 20.04 here. I need help too.

**Comment by soultaker6** [comment]
In my case it doesn't get to the logo part. The game stops when steam loading screen ends, but the funny thing is that in  steam console I can see that dota is loading and then it says running for a second or so (but dota isn't booted even) after that happens it goes back like nothing had happened. I've tried many different thing to this point and I still can't fix this.
